components/accounts/utils.ts

```diff
@@ -18,7 +18,7 @@ export const toDistribute = (length: number, distribution: number): number => {
 }
 
 export const parseBatchAddresses = (batchAddresses: string): string[] => {
-  const ss58Format: any = store.getters.getChainProperties58Format
+  const ss58Format: any = store.getters['chain/getChainProperties58Format']
```

Comment:
is this like a good pattern for that ?

Reply:
Oups! I'm used to this [store](https://nuxtjs.org/docs/directory-structure/store/) pattern
```js
  const ss58Format: any = store.getters.chain.getChainProperties58Format
```
is working too, should I change it?